packages/reporter/cypress/integration/tests_spec.ts

```diff
@@ -103,5 +103,9 @@ describe('controls', function () {
         .should('be.visible')
       })
     })
+
+    it('command name is not uppercase', () => {
+      cy.get('.command-name-visit .command-method').should('not.have.css', 'text-transform', 'uppercase')
```

Comment:
Not really sure this test is necessary, but we've increasingly been running into difficulty in  how we test the runner/reporter CSS changes lately.

Reply:
I decided to remove this test. Because it cannot alarm when the visual change happens. [Percy](https://percy.io/) or [Chromatic](https://www.chromatic.com/) would be the better solution than Cypress for visual changes.